src/Tribe/Admin/Notices.php

```diff
@@ -237,4 +240,32 @@ public function maybe_display_plus_commerce_notice() {
 			tribe_notice( "event-tickets-plus-missing-{$provider}-support", $message, 'dismiss=1&type=warning' );
 		}
 	}
+
+	/**
+	 * Display notices for unsupported currencies.
+	 *
+	 * @since 5.5.5
+	 *
+	 * @return void
+	 */
+	public function maybe_display_unsupported_currency_notice() {
+		if ( Currency::is_supported_currency() ) {
+			return;
+		}
+
+		$message = sprintf(
+			'<h3>%1$s</h3><p>%2$s</p>',
+			esc_html( Currency::$unsupported_currency['heading'] ),
+			esc_html( Currency::$unsupported_currency['message'] )
+		);
+
+		tribe_notice(
+			'event-tickets-unsupported-currencies',
```

Comment:
We can maybe also add the currency symbol to the notice ID, so that if they change in the future (and the notice was dismissed) we can have the notice showing.